components/bpmn/bpmn-explorer-web/src/web/model/advancedFilterModel.jag

```diff
@@ -0,0 +1,237 @@
+                          vCollection.variables.push({
+                            "name":variableNames[i],
+                            "value":values[i],
+                            "operation" : "equals"
+                        });
+                               
+                    } else {
+                         break;
+                    }
+                }
+            }             
+           
+        }
+
+        vCollection["start"] = ((pagination - 1) * pageSize);
+        vCollection["size"] = pageSize;
+        vCollection["finished"] =  "true";
+        vCollection["tenantId"] =  bpsTenantID;
+        bodyOfRequest= vCollection;          
+     
+        requestUrl = httpUrl + "/bpmn/query/historic-process-instances";
+       
+
+    } else {
+            
+        if (requestParameters.processInstanceId !== undefined && requestParameters.processInstanceId.length > 0) {
+           
+            vCollection["processInstanceId"] = requestParameters.processInstanceId;
+        }
+
+        if (requestParameters.processDefinitionNameLike !== undefined && requestParameters.processDefinitionNameLike.length > 0) {
+             
+            vCollection["processDefinitionId"] = requestParameters.processDefinitionNameLike;         
+        }      
+     
+       if (requestParameters.variableName !== undefined && requestParameters.variableName.length > 0 && 
+            requestParameters.variableValue !== undefined && requestParameters.variableValue.length > 0) {
+
+            var variableNames = requestParameters.variableName;        
+            var values = requestParameters.variableValue;
+            vCollection = {"variables":[]};
+            if (typeof variableNames == 'string') {
+
+                vCollection.variables.push({
+                            "name":variableNames,
+                            "value":values,
+                            "operation" : "equals"
+                });
+
+            } else {            
+
+                for (var i = 0; i < variableNames.length; i++) {
+
+                    if (variableNames[i] != null && variableNames[i] !== "") {                  
+                       
+                          vCollection.variables.push({
+                            "name":variableNames[i],
+                            "value":values[i],
+                            "operation" : "equals"
+                        });
+                               
+                    } else {
+                         break;
+                    }
+                }
+            }        
+                        
+        }
+
+        vCollection["start"] = ((pagination - 1) * pageSize);
+        vCollection["size"] = pageSize;
+        vCollection["tenantId"] =  bpsTenantID;
+        bodyOfRequest= vCollection;  
+       
+        requestUrl = httpUrl + "/bpmn/query/process-instances";      
+    }
```

Comment:
The above method has lots of empty new lines. To avoid readability issues, please dont keep new lines as done on line 165, 167, 169, 171, 177, 183.